bedrock/foundation/templates/foundation/base-resp.html

```diff
@@ -44,8 +54,32 @@
         </li>
       {% endfor %}
       </ul>
+      <ul class="nav">
+        <li><b>Foundation Programs</b></li>
```

Comment:
Bah. I double checked that I wrapped all the strings but still somehow knew I'd miss something  😅